pos_reset_search/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Did you use the [readme-generator](https://github.com/OCA/maintainer-tools/blob/master/README.md#readme-generator)  to generate this file? I do not see the readme directory.

Reply:
I did not use the readme generator. The `oca-gen-addon-readme` tool is not producing anything and gives zero error message or useful information.

Reply:
You need to have at least `readme/DESCRIPTION.rst` in your module, you could also have `readme/CONTRIBUTORS.rst` and `readme/USAGE.rst` in this case.

Then from your module history, run:
```
oca-gen-addon-readme --repo-name pos --branch 9.0 --addon-dir .
```

Reply:
Oh! much better this way thank you for the walkthrough.